torch/_inductor/dependencies.py

```diff
@@ -189,7 +189,7 @@ def reads_and_writes(self):
         return itertools.chain(self.reads, self.writes)
 
 
-class _RecordLoadStoreInner(V.MockHandler):
+class _RecordLoadStoreInner(V.MockHandler):  # type: ignore[name-defined]
```

Comment:
not clear to me why mypy thinks V.MockHandler is undefined in this context -- yet it's fine with e.g. a `foo = V.MockHandler` statement